config/memes.php

```diff
@@ -735,4 +735,10 @@
         'text' => ':cooljojo:',
         'public' => true,
     ],
-];
\ No newline at end of file
+    [
+        'code' => ':oranhead:',
+        'image' => "<img src='./images/memes/oran_head.png' />",
+        'text' => ':oranhead:',
+        'public' => true,
+    ],      
```

Comment:
```suggestion
```
This file doesn't need to be updated anymore and a merge conflict will block it, let's drop these changes 👍